### PR TITLE
feat: requested book list integration

### PR DESCRIPTION
**Pull request by zhpeng811**
showing a list of requested books requested by current user

### CODE REVIEW

File: app/src/main/java/com/cmput301f20t21/bookfriends/repositories/RequestRepository.java [hunk 1]
```diff
@@ -46,8 +46,9 @@ public static IRequestRepository getInstance() {
                 .whereEqualTo("status", REQUEST_STATUS.BORROWED.toString()).get();
     }
 
-    public Task<List<Request>> getAllRequestsByUsername(String username) {
-        return requestCollection.whereEqualTo("requester", username).get()
+    public Task<List<Request>> getAllRequestsByUsername(String username, REQUEST_STATUS status) {
+        return requestCollection.whereEqualTo("requester", username)
+                .whereEqualTo("status", status.toString()).get()
```
Comment (zhpeng811) [inline app/src/main/java/com/cmput301f20t21/bookfriends/repositories/RequestRepository.java:51]: this function can replace the getBorrowedRequestByUsername() above, whoever will be working on borrowed page can change that

File: app/src/main/java/com/cmput301f20t21/bookfriends/repositories/RequestRepository.java [hunk 2]
```diff
@@ -46,8 +46,9 @@ public static IRequestRepository getInstance() {
                 .whereEqualTo("status", REQUEST_STATUS.BORROWED.toString()).get();
     }
 
-    public Task<List<Request>> getAllRequestsByUsername(String username) {
-        return requestCollection.whereEqualTo("requester", username).get()
+    public Task<List<Request>> getAllRequestsByUsername(String username, REQUEST_STATUS status) {
```
Comment (c25vdw) [inline app/src/main/java/com/cmput301f20t21/bookfriends/repositories/RequestRepository.java:49]: maybe rename this to getRequestsByUsernameAndStatus or just getRequests? currently the arguments do not match the method name.

**Review by c25vdw — APPROVED** [review]
lgtm. renaming is not necessary right now.

**Review by QuocTrungTran — APPROVED** [review]
lgtm

**Pull request merged**